swagger.yaml

```diff
@@ -3113,6 +3113,15 @@ paths:
         - userAuth: [admin, read-only-admin, user]
       operationId: v2ListSupportedOpenshiftVersions
       description: Retrieves the list of OpenShift supported versions.
+      parameters:
+        - in: query
+          name: version_pattern
+          description: Version pattern for the returned releases. E.g. '4.12'.
```

Comment:
You would literally put `4.12` in here? Is that how that works?

Would it be better for this to be an enum with `X.Y` and `X.Y.Z` as members or are there more options?

Reply:
Currently it retrieves every release image such that `version_pattern` is substring of `ReleaseImage.Version`

Reply:
Okay if we want this then we should change the name.
I'd call it just `version` assuming it will return all the data for the versions matching whatever is sent.